src/globus_cli/login_manager/tokenstore.py

```diff
@@ -10,15 +11,218 @@
 from .scopes import CURRENT_SCOPE_CONTRACT_VERSION
 
 if t.TYPE_CHECKING:
-    from globus_sdk.tokenstorage import SQLiteAdapter
+    from globus_sdk.tokenstorage import SQLiteAdapter  # noqa: F401
 
 # env vars used throughout this module
 GLOBUS_ENV = os.environ.get("GLOBUS_SDK_ENVIRONMENT")
 
 
-# stub to allow type casting of a function to an object with an attribute
-class _TokenStoreFuncProto:
-    _instance: SQLiteAdapter
+class CLITokenstorage:
+    """
+    A wrapper over the globus-sdk's v1 tokenstorage which provides simplified
+    capabilities specific to the CLI.
+    """
+
+    def __init__(self) -> None:
+        self._adapter: SQLiteAdapter | None = self._construct_adapter()
+
+    def _construct_adapter(self) -> SQLiteAdapter:
+        from globus_sdk.tokenstorage import SQLiteAdapter  # noqa: F811
```

Comment:
Some local testing showed that this import statement takes about 80ms; is that too high of a penalty?

Reply:
We discussed! In the past, `globus_sdk.tokenstorage` was part of the "bad path" for eventually importing `requests`, but we fixed that. So now I'm going to let the eager import happen and we should be fine. :smile: